Tiltfile

```diff
@@ -1,6 +1,20 @@
 # -*- mode: Python -*-
 
 load("ext://uibutton", "cmd_button", "text_input")
+envsubst_cmd = "./hack/tools/bin/envsubst"
```

Comment:
Let's drop the diff from this PR. The release-1.2 Tiltfile worked quite a bit differently and it's not super important that we use 1.26 there